docs/cron-workflows.md

```diff
@@ -108,6 +109,36 @@ For example, with `timezone: America/Los_Angeles`:
     |            | 2        | 2020-11-02 02:01:00 -0800 PST |
     |            | 3        | 2020-11-03 02:01:00 -0800 PST |
 
+#### Skip forward
+
+You can use `minimumInterval` to schedule once per day, even if the time you want is in a daylight saving skip forward period where it would otherwise be scheduled twice.
+
+An example 02:30:00 schedule
+
+```yaml
+schedules:
+  - 30 2 * * *
+  - 0 3 * * *
+minimumInterval: 60m
+```
+
+The 3:00 run of the schedule will not be scheduled every day of the year except on the day when the clock leaps forward over 2:30.
+In that case the 3:00 run will run.
+
+#### Skip backwards (duplication)
+
+You can use `minimumInterval` to schedule once per day, even if the time you want is in a daylight saving skip backwards period where it would otherwise not be scheduled.
+
+An example 01:30:00 schedule
+
+```yaml
+schedule: 30 1 * * *
+minimumInterval: 120m
+```
+
+This will schedule at the first 01:30 on a skip backwards change.
+The second will not run because of `minimumInterval`.
+
```

Comment:
```suggestion
#### `minimumInterval`

You can set a minimum interval between runs.

For example, if you want to ensure you only run once a day during a DST leap forward:

```yaml
schedules:
  - 30 2 * * *
  - 0 3 * * *
minimumInterval: 60m
\```

The 3:00 schedule will never run except on the day the clock leaps forward over 2:30.
In that case, only the 3:00 schedule will run and the 2:30 will not.

Another example, if you want to ensure you only run once a day during a DST leap backward:

```yaml
schedule: 30 1 * * *
minimumInterval: 120m
\```

The first run will be at 01:30.
On the day the clock leaps backward, the second will not run because of `minimumInterval`.

```

- I think we can simplify this a bit and combine the two sections as two examples. We can also reduce the verbal descriptions as the examples are a lot clearer than them (I found the initial descriptions to be a bit confusing and verbose. The descriptions afterward are good, though I simplified those as well)
- Use consistent language with "runs" vs. "schedule", and "leap" vs "skip"
  - "leap" has a less technical terminology and is used often for DST (and infrequently for non-DST), so I think works a bit better as specific to DST

Reply:
I didn't like making this a single document. In the skim reading case you MUST only care about one or the other per cronworkflow, so have kept the two items separate.

I have also kept the description verbose as they are complex to understand I believe it's helpful for users to understand how these work rather than just use them as magic words.

Reply:
> In the skim reading case

> I have also kept the description verbose as they are complex to understand I believe it's helpful for users to understand how these work

These two are mutually exclusive scenarios. Either you can skim read it or it's verbose, and therefore you cannot skim read it. To understand how it works, you also would not be skim reading.

> In the skim reading case

In the skim reading case, you'd just search "DST" rather than _which_ transition. 
But I don't think we should be tailoring docs to skim readers in any case, that's not a good way of building depth, and as per above, you can really only choose one or the other.

> I have also kept the description verbose as they are complex to understand I believe it's helpful for users to understand how these work 

I don't think they're complex to understand IMO, and the verbosity actually makes it _harder_ to understand. More words is not necessarily a good thing, especially when the words are inconsistent. The style guide says to use [simplicity and directness](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language) for a reason, after all. And exceptions to the style guide lead to bikeshedding ad infinitum.

> rather than just use them as magic words.

There's a Wikipedia link provided. We don't define the term "leap", it already exists. The same way we don't define the term "Pod" and rather link to the k8s docs.

Reply:
Followed up with https://github.com/argoproj/argo-workflows/pull/13723#discussion_r1805617431